SSS/README.txt

```diff
@@ -0,0 +1,13 @@
+test_move_anon_raw.fif
+Raw data, but magnetometers with channel type '3022' were force to be T3 with channel type '3024'.
+
+test_move_anon_raw_sss.fif
+MaxFilter params: '-in 8 -out 3'
```

Comment:
I don't think you need to specify these, they are defaults. Probably best just to list the MaxFliter version used (2.2) at the top of the file, and omit otherwise default params.